internal/util/fs/fs.go

```diff
@@ -1,28 +1,29 @@
-package source
+package fs
 
 import (
 	"errors"
 	"fmt"
+	"io/fs"
 	"os"
 	"path/filepath"
 	"time"
 )
 
 const (
-	OwnerWritableFileMode os.FileMode = 0700
-	OwnerWritableDirMode  os.FileMode = 0700
-	OwnerReadOnlyFileMode os.FileMode = 0400
-	OwnerReadOnlyDirMode  os.FileMode = 0500
+	ownerWritableFileMode os.FileMode = 0700
```

Comment:
I wonder if we should export these and use them in the image utils. Maybe we could even add a comment here to explain why we don't give group perms.

Reply:
I didn't see any usage outside of this package, so opted to unexport them. If there's a strong use case that comes up later, we can always export them again. This is all internal, so it doesn't matter either way from a public API standpoint, but it can be nice to keep even internal packages nice and tidy to avoid leaky abstractions.